environments/sso_vars.yaml

```diff
@@ -0,0 +1,30 @@
+# Parameters for single-signon.yaml and enable-federation-openidc.yaml
+parameter_defaults:
+  KeystoneTrustedDashboards: https://esi.massopen.cloud/
+  # Copy the IdP name from moc openstack. Should we use a different provider for ESI?
+  KeystoneOpenIdcIdpName: moc
```

Comment:
This is just a local label for the remote provider. You can call it anything you want; `moc` seems appropriate.